v_babiichuk/variables-and-types/city.py

```diff
@@ -12,3 +12,7 @@
 # Я народився в місті Львів. Його населення - приблизно 721301 людей, а площа - 182.8 км².
 # ```
 # Примітка: Замініть "Львів", "721301" та "182.8" на власні значення, які ви ввели.
+city = "Бородянка"
+population = str("13157")
+area = str("6,5")
```

Comment:
"13157" і "6,5" можна не перетворювати в строку, так як вони вже є строками


```suggestion
population = "13157"
area = "6,5"
```

Reply:
Через f виводить чи так залишається код же працює.

Reply:
Пусть остается